index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/266f4d4d90.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="nav-container">
+        <div class="wrapper">
+            <nav>
+                <div class="logo">
+                    JONAH.
+                </div>
+                <div class="mobile-menu">
+                    <i class="fas fa-bars"></i>
+                </div>
+            </nav>
+        </div>
+    </div>
+
+    <div class="header-container-mobile">
+        <div class="wrapper">
+            <header>
+                <div class="intro">
+                    <h1>I'm Jonah <br>Glad to see you!</h1>
+                    <p>
+                        I’m a software developer! I can help you build a product feature or website. Look through some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate to contact me.
+                    </p>
+                    <a href="#">
+                        LET'S CONNECT
+                    </a>
+                </div>
+            </header>
+
+            <div class="social-icons">
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-facebook"></i>
+                <i class="fab fa-linkedin-in"></i>
+                <i class="fab fa-github"></i>
+                <i class="fab fa-youtube"></i>
+            </div>
```

Comment:
- [ ] Kindly change your `div` tag to a `ul` tag and wrap each of your `icons` in an `li` tag so your code can be semantic which would also make your site better for screen readers. You can read more about semantic HTML in this [article](https://www.w3schools.com/html/html5_semantic_elements.asp) 🙏